xwiki-platform-core/xwiki-platform-annotation/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/Script.xml

```diff
@@ -560,1365 +560,1365 @@ return XWiki;
       <cache>long</cache>
     </property>
     <property>
-      <code>## Retrieve the annotation settings from the configuration object
-#set($config = 'AnnotationCode.AnnotationConfig')
-#set($configObj = $xwiki.getDocument($config).getObject($config))
-#set($annotationHighlightByDefault = $configObj.getProperty('displayHighlight').value)
-#set($annotationsDisplayedByDefault = $configObj.getProperty('displayed').value)
-#set($annotationsActivated = $configObj.getProperty('activated').value)
-#set($exceptionSpaces = $configObj.getProperty('exceptionSpaces').value)
-#set($annotationClass = $configObj.getProperty('annotationClass').value)
+      <code>// Retrieve the annotation settings from the configuration object
+const config = JSON.parse(document.getElementById('annotation-config').text);
+
+define('xwiki-annotation-messages', {
+  prefix: '',
+  keys: [
+    'annotations.annotated.error.noannotatedelement',
+    'annotations.annotated.error.wrongsyntax',
+    'annotations.menu.loading',
+    'annotations.menu.loaderror',
+    'annotations.annotated.loading',
+    'annotations.annotated.loaderror',
+    'annotations.annotated.loaderror.wrongresponse',
+    'annotations.action.delete.confirm',
+    'annotations.action.delete.inProgress',
+    'annotations.action.delete.done',
+    'annotations.action.delete.failed',
+    'annotations.action.validate.success',
+    'annotations.action.validate.loaderror',
+    'annotations.action.edit.form.loaderror',
```

Comment:
Note: this key was introduced by https://github.com/xwiki/xwiki-platform/commit/3847fd1008ad648a9a1f8cf21b732afea0035c23 in 2010 and never had a corresponding translation key.
We need to define one.